### ISSUE TITLE
Projects: Trying to follow basic setup and failing

### ISSUE THREAD

**Issue by danielloader** [squidfunk/mkdocs-material#6360]
### Context

- Python 3.11.6
- MacOS 14.1
- Mkdocs 1.5.1

### Bug description

I've tried to follow the project plugin documentation to set up a basic example, with a single example project inside the `projects` directory.

However doing the bare minimum to try and get this working has ended in failure for me.

### Related links

- [Reporting a bug](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/)
- [Projects Plugin](https://squidfunk.github.io/mkdocs-material/plugins/projects/))
- [Projects Plugin Roadmap Discussion](https://github.com/squidfunk/mkdocs-material/discussions/5800)

### Reproduction

[9.4.9+insiders.4.43.0-projects-missing-directory.zip](https://github.com/squidfunk/mkdocs-material/files/13401107/9.4.9%2Binsiders.4.43.0-projects-missing-directory.zip)

### Steps to reproduce

1. Create initial parent mkdocs project using `mkdocs new .`
2. Configure resulting `mkdocs.yml` file with theme being set to material, projects plugin enabled.
3. Create `projects` directory in the root of the project.
4. Execute `mkdocs new abc` in the `projects` directory.
5. Add relevant navigation to link the two using the `project://` url scheme.
    ```yaml
    nav:
      - Home: index.md
      - abc: project://abc
    ```
6. Try and run `mkdocs serve`.
7. Get the resulting error:
    ``` shell
    > mkdocs serve
    INFO    -  Building documentation...
    INFO    -  Cleaning site directory
    ERROR   -  Couldn't find project '.abc'
    
    Aborted with a BuildError!
    ```

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/).
- [X] I have attached links to [the documentation](https://squidfunk.github.io/mkdocs-material/), and possibly related [issues](https://github.com/squidfunk/mkdocs-material/issues) and [discussions](https://github.com/squidfunk/mkdocs-material/discussions).
- [X] I assure that I have [removed all customizations](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/#remove-customizations) before submitting this bug report.
- [X] I have attached a __.zip file__ with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/).

**Comment by squidfunk** [comment]
Thanks for reporting! Unfortunately, the info plugin does not consider the projects folder, so it's not contained. This is a bug in the info plugin that we should fix. Could you please zip up the contents manually and reattach them here?

**Comment by squidfunk** [comment]
Okay, I think I know what's happening: if you create a project using `mkdocs new abc`, the new project will not have `theme.name: material` set. However, the projects plugin needs to add itself to the nested project, but since the theme is not `material`, the plugin cannot be resolved. This is the reason why you see the error as reported in https://github.com/squidfunk/mkdocs-material/discussions/5800#discussioncomment-7607307:

> Strange, I get a different error message: ERROR   -  [.a] Config value 'plugins': The "projects" plugin is not installed. I am using insiders and using the same venv for building the [examples](https://github.com/mkdocs-material/examples) works fine.

If after creation, I set `theme.name: material`, the plugin works as expected. I attached a working version of this. If MkDocs would add an option to specify the name of the theme when running `mkdocs new`, that would be really amazing. Maybe this is something to propose to the maintainers of MkDocs.

[projects-example.zip](https://github.com/squidfunk/mkdocs-material/files/13402687/projects-example.zip)

**Comment by squidfunk** [comment]
[`31a00bb`](https://github.com/squidfunk/mkdocs-material-insiders/commit/31a00bb7bc3b171f5188d2274bd6e05b9c483bf0) fixes the dependence of the projects plugin on Material for MkDocs. The projects plugin now works with any theme, and you can also mix themes in nested projects. Just remember to fully qualify the plugin with `material/projects`:

``` yaml
plugins:
  - material/projects
```

__Result__

![screenshot-localhost-3000-1700383495106](https://github.com/squidfunk/mkdocs-material/assets/932156/4ac10b82-243c-4f56-a0b5-e3abb1540e0a)

**Comment by danielloader** [comment]
Perfect, your example zip works as expected - and while I don't think I'd use this functionality without the theme it's appreciated anyway.

**Comment by squidfunk** [comment]
We'll keep it open until released.

**Comment by squidfunk** [comment]
> Perfect, your example zip works as expected - and while I don't think I'd use this functionality without the theme it's appreciated anyway.

The main idea is to improve the editing experience, so `mkdocs new abc` does not crash ☺️

**Comment by squidfunk** [comment]
Released as part of [9.4.10+insiders-4.43.1](https://squidfunk.github.io/mkdocs-material/insiders/changelog/#4.43.1).
